File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        ans =[]
        n = len(nums)
        li= [0]*(n+1)
        for i in range (n):
            li[nums[i]]+=1
        for i in range (1,n+1):
            if li[i]==0:
                ans.append(i)
        return ans


        

File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        int n = nums.length;
        ArrayList<Integer>list = new ArrayList<>();
        int frq[]= new int [n+1];
        for ( int i =0;i<n;i++){
            frq[nums[i]]++;
        }
        for ( int i =1;i<n+1;i++){
            if (frq[i]==0){
                list.add(i);
            }
        }
        return list;

        

    }
}